tools/ci_build/github/azure-pipelines/templates/component-governance-component-detection-steps.yml

```diff
@@ -20,17 +20,20 @@ steps:
     inputs:
       # ignore unit tests in emscripten. emscripten unit tests are not used in onnxruntime build
       # ignore onnx third_party directory. onnx third_party directory is not responsible for onnxruntime build
-      # because sometime there are multiple repo is checked out in the same pipeline, we also need to add the repo name, such as onnxruntime/ to the ignoreDirectories
+      # ignore pybind11 third_party directory. pybind11 third_party directory is not responsible for onnxruntime build
+      # ignore pybind11 tests directory. pybind11 tests directory is not responsible for onnxruntime build
       # ignore onnxruntime-extensions directory. onnxruntime-extensions directory is not responsible for onnxruntime build
       # ignore react_native e2e node_modules directory. react_native e2e node_modules directory is generated by react_native e2e tests
-      # ignore github directory. github directory is used for github actions, not for onnxruntime package
+      # ignore js node_modules directory. js node_modules directory is generated by js tests
       # ignore onnxruntime-inference-examples directory. onnxruntime-inference-examples directory is used for inference examples, not for onnxruntime package
       # ignore BinariesDirectory. BinariesDirectory is used for build output, not for onnxruntime package
       ignoreDirectories:
         '$(Build.Repository.LocalPath)/cmake/external/emsdk/upstream/emscripten/tests,
         $(Build.Repository.LocalPath)/cmake/external/onnx/third_party/benchmark,
         $(Build.Repository.LocalPath)/cmake/external/onnx/third_party/pybind11,
+        $(Build.Repository.LocalPath)/cmake/external/onnx/third_party/pybind11/tests,
         $(Build.Repository.LocalPath)/cmake/external/onnxruntime-extensions,
         $(Build.Repository.LocalPath)/js/react_native/e2e/node_modules,
```

Comment:
Why the node_modules are ignored?

Reply:
They are not part of the source code. They are generated.

Reply:
Generated by what? Should we monitor what nodejs modules are used in our pipelines?

Reply:
Are these files get distributed with our web and React Native packages?

Reply:
No, from Yulong